apps/src/applab/designMode.js

```diff
@@ -216,7 +216,18 @@ designMode.onPropertyChange = function(element, name, value) {
       element.setAttribute('rows', value);
       break;
     default:
-      throw "unknown property name " + name;
+      // Mark as unhandled, but give typeSpecificPropertyChange a chance to
+      // handle it
+      handled = false;
+  }
+
+  if (elementLibrary.typeSpecificPropertyChange(element, name, value)) {
+    designMode.editElementProperties(element);
+    handled = true;
```

Comment:
Yeah, I guess right now there are two things we care about from child handlers. (1) Does the child know about this type, such that we can consider it a known property and (2) Did the child change something, such that the element properties need to be updated.
Right now those two things are always in sync, but it's not unreasonable to think that they might not be in the future. I don't see a super clean way to handle this (could have a separate function to look to see if the elementLibrary will handle the type, or could have the function return a pair of bools), so I think I'm inclined to leave this be until we need a situation where the two things are not in sync.